src/flownet/templates/observations.ertobs.jinja2

```diff
@@ -24,6 +24,9 @@ SUMMARY_OBSERVATION WGPR_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.gas
 {%- if not isnan(kw.water_rate) and error_config.WWPR.rel_error is not none and error_config.WWPR.min_error is not none: %}
 SUMMARY_OBSERVATION WWPR_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.water_rate }}; ERROR = {{ [error_config.WWPR.rel_error * kw.water_rate, error_config.WWPR.min_error] | max }}; DATE  = {{ date_formatted  }}; KEY = WWPR:{{ kw.well_name }}; };
 {%- endif %}
+{%- if not isnan(kw.salt_rate) and error_config.WSPR.rel_error is not none and error_config.WSPR.min_error is not none: %}
```

Comment:
could also add possibility for salt_total / WSPT here for completeness, but i guess none will ever use it...